CyanEmu/Scripts/CyanEmuPlayerController.cs

```diff
@@ -504,21 +505,23 @@ public void EnterStation(CyanEmuStationHelper station)
                 currentStation_.ExitStation();
             }
 
-            currentStation_ = station;
-
             if (!station.IsMobile)
             {
                 characterController_.enabled = false;
+                station.EnterLocation.Translate(0f, -SITTING_HEIGHT_ADJUSTMENT_, 0f);
```

Comment:
I'm concerned with moving the station's enter location directly. Since the enter location is just another transform, it could be moved through animation or other script and this has potential to mess that up. Could you change this to instead update the player's stance? This would move the camera up or down if they are in a station and not modify external objects.